Makefile

```diff
@@ -24,6 +24,7 @@
 CC       = gcc
 AR       = ar
 CPPFLAGS =
+#CFLAGS   = -g -Wall -O2 -pedantic -std=c99 -D_XOPEN_SOURCE=600
```

Comment:
It's more of a self document as to how to enable strict C99 mode.  I think it adds value as a comment personally.